website/content/en/docs/best-practices/best-practices.md

```diff
@@ -40,6 +40,8 @@ Considerations for Operator developers:
 
 - Operators may create objects as part of their operational duty. Object accumulation can consume unnecessary resources, slow down the API and clutter the user interface. As such it is important for operators to keep good hygiene and to clean up resources when they are not needed. Here are instructions on [how to handle cleanup on deletion][advanced-topics].
 
+- You can leverage Knative Service to optimize the resource usage for your operands. Refer to [Converting a Kubernetes Deployment to a Knative Service](https://knative.dev/docs/serving/convert-deployment-to-knative-service/) for detail.
+
```

Comment:
Thank you for your contribution. However, I am not sure if https://knative.dev/docs/serving/convert-deployment-to-knative-service would fit as best practices here. This doc is more about general good practices to work with operators. So, why Converting a Kubernetes Deployment to a Knative Service can be classified as common good practice?

Reply:
With Knative, we can dynamically scale the operands for operators, even scale to 0, this will help a lot for a system which was composed by tons of operators.

I also have a blog here https://medium.com/ibm-cloud/using-serverless-model-to-reduce-footprint-for-your-kubernetes-openshift-platform-d06bfbeb65f2 clarifying, but was hoping we can claim this in official document so that it can help more people who want to optimize their cluster resources.

Reply:
That seems a nice cool feature, however, the scope of this doc is provided to the end-users best practices and in this way would be for e.g. (not saying that it is a best practice) ensure that you are able to dynamically scale operands for operators, even scale to 0 when `<condition>` and `<why>`. 

I mean, I do not think that adapted any third-party dep could fit here as a best practice.  Indeed the same goal probably can be achieved in other ways so beyond the Knative can be a very nice project I cannot go along that its adoption could/should be considered a best practice to write Operators.